src/bin/identities/CommandAuthenticate.ts

```diff
@@ -45,32 +45,21 @@ class CommandAuthenticate extends CommandPolykey {
         });
         const providerMessage = new identitiesPB.Provider();
         providerMessage.setProviderId(providerId);
-        providerMessage.setMessage(identityId);
-        const successMessage = await binUtils.retryAuthentication(
-          async (auth) => {
-            const stream = pkClient.grpcClient.identitiesAuthenticate(
-              providerMessage,
-              auth,
-            );
-            const codeMessage = (await stream.next()).value;
-            process.stdout.write(
-              binUtils.outputFormatter({
-                type: options.format === 'json' ? 'json' : 'list',
-                data: [`Your device code is: ${codeMessage!.getMessage()}`],
-              }),
-            );
-            return (await stream.next()).value;
-          },
-          meta,
-        );
-        process.stdout.write(
-          binUtils.outputFormatter({
-            type: options.format === 'json' ? 'json' : 'list',
-            data: [
-              `Successfully authenticated user: ${successMessage!.getMessage()}`,
-            ],
-          }),
-        );
+        providerMessage.setIdentityId(identityId);
+        await binUtils.retryAuthentication(async (auth) => {
+          const stream = pkClient.grpcClient.identitiesAuthenticate(
+            providerMessage,
+            auth,
+          );
+          const codeMessage = (await stream.next()).value;
```

Comment:
Are you supposed to take 1 value from the stream? Why is it stream at all then?

Reply:
DW about this. This is because I think alot of CLI calls are bundling too much logic within the `retryAuthentication` lambda. It should only be done on the initial call. All stream related work then happens outside and afterwards.

Reply:
This would need to be resolved in a later PR. New issue for this.

Reply:
Previously it was returning the identity id as the second value, which seemed to be unecessary considering it's a parameter for the command. I kept it as a stream after doing that since the authenticate function on the provider is itself a stream so I wasn't sure if it would still work as a unary call.

Reply:
`identities authenticate` is now a unary call. Still works as expected with all tests passing.